src/core/control/tools/VerticalToolHandler.cpp

```diff
@@ -108,12 +113,12 @@ void VerticalToolHandler::paint(cairo_t* cr) {
     cairo_set_source_surface(cr, this->crBuffer, 0, elemY);
     cairo_paint(cr);
 
-#ifdef DEBUG_SHOW_PAINT_BOUNDS
-    cairo_rectangle(cr, 0, elemY, cairo_image_surface_get_width(this->crBuffer),
-                    cairo_image_surface_get_height(this->crBuffer));
-    cairo_set_source_rgba(cr, 1.0, 0, 0, 0.3);
-    cairo_fill(cr);
-#endif
+    IF_DEBUG_REPAINT({
+        cairo_rectangle(cr, 0, elemY, cairo_image_surface_get_width(this->crBuffer),
+                        cairo_image_surface_get_height(this->crBuffer));
+        cairo_set_source_rgba(cr, 1.0, 0, 0, 0.3);
+        cairo_fill(cr);
+    })
```

Comment:
nit: add a semicolon to avoid weird editor line breaking behavior (e.g., to pretend this is a statement).